NitroxPatcher/Patches/Dynamic/SubFire_EngineOverheatSimulation_Patch.cs

```diff
@@ -0,0 +1,23 @@
+# if DEBUG
+using System.Reflection;
+using NitroxModel.Helper;
+
+namespace NitroxPatcher.Patches.Dynamic;
+
+/// <summary>
+/// Disables cyclops overheating from moving while in creative mode (really annoying for no reason)
+/// </summary>
+public sealed partial class SubFire_EngineOverheatSimulation_Patch : NitroxPatch, IDynamicPatch
```

Comment:
Don't think it's a good idea to have this one, besides changing the behavior it may cause some desync if the driver is in creative & the passengers are not

Reply:
There's a DEBUG if tho is that not enough ? And this subsystem isn't synced yet I believe so it doesn't matter if it gets desynced during a DEBUG test

Reply:
I think would be good to have a log when this debug feature is active. 
Ideally we'd have a "dev patches" project that isn't committed with all the hacks we want during testing.